public/input.css

```diff
@@ -88,12 +88,28 @@ a {
 
 .keyboard {
   user-select: none;
-  display: flex;
-  flex-wrap: wrap;
-  gap: 6px;
-  justify-content: center;
-  margin: 0 auto 8px;
-  font-size: 1.5rem;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  gap: 0.25rem;
+}
+
+/* Kind of redundant, here for completeness */
+.keyboard > button:nth-child(-1n + 12) {
```

Comment:
Docs on nth-child functional notation: https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#functional_notation

tl;dr - This tells the first 12 elements they should render on the first row. 

Note: because CSS prefers whichever styles are written last the order of these classes  matters here.